packages/cordova/template/package.json

```diff
@@ -19,7 +19,7 @@
   "private": true,
   "repository": "",
   "enact": {
-    "theme": "moonstone",
+    "theme": "sandstone",
```

Comment:
can you also update the "version" mentioned above?
Let's put version 2 and then see other opinions

Reply:
I believe version number is for the user when starting a new project using this template, so it should be 1.0.0.